pyproject.toml

```diff
@@ -151,6 +154,8 @@ extend-ignore = [
 'src/trio/lowlevel.py' = ['F401']
 'src/trio/socket.py' = ['F401']
 'src/trio/testing/__init__.py' = ['F401']
+# Don't check annotations in notes-to-self
+'notes-to-self/*.py' = ['ANN']
```

Comment:
If we're going to do this, then maybe restore `notes-to-self/*.py` to what they were before (I *think* the correct invocation is `git reset main -- notes-to-self`?)

Reply:
+1

Reply:
Restored in 12fe0343